<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsBox</class>
 <widget class="QWidget" name="SettingsBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>440</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SettingsBox</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>441</height>
    </rect>
   </property>
   <widget class="QLabel" name="schemeLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="schemeName">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Default scheme</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="schemeUse">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>use scheme</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>390</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="progressLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>progress:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="loadScheme"/>
    <addaction name="saveScheme"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="loadScheme">
   <property name="text">
    <string>Load scheme</string>
   </property>
  </action>
  <action name="saveScheme">
   <property name="text">
    <string>Save scheme</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
